src/gui/mainwindow.cpp

```diff
@@ -1514,6 +1515,22 @@ void MainWindow::loadPreferences(bool configureSession)
     }
 #endif
 
+    {
+        static QFileSystemWatcher *styleSheetWatcher = nullptr;  // watches stylesheet for changes to give instance feedback
```

Comment:
Tracking changes in the style file is too much!.. Dissaproved.

Reply:
Helps when making new style file, helped me myself when i was testing this.

Think about the designer, don't have to restart qbittorrent...

Though I am ready to remove it if you insist

Reply:
My belief is that minor features that are not directly related to the main purpose of the application should be implemented as less expensive as possible. Interface customizing is minor feature and "Helps when making new style file, helped me myself when i was testing this" is even more minor (qBittorrent isn't Style design studio).
>Think about the designer, don't have to restart qbittorrent again and again

Reloading the style file on demand (when applying application settings) is more than enough.